signup/signupController.js

```diff
@@ -5,6 +5,10 @@ define('signupController', [], function() {
 		$scope.sharedProperties = sharedPropertiesService.getSharedProperties();
 		$scope.successMessage = null;
 		$scope.appointmentId = null; // The id of the client's appointment once they successfully sign up
+		$scope.hasExistingAppointment = null;
```

Comment:
Does this actually need to be in `$scope`? It seems like it's only used in the sign-up method

Reply:
You are right, I initially had it here but I ended up throwing all into the storeAppointment method to make the async stuff simpler lol. This $scope variable is a relic of my inexperienced attempts at implementing this functionality. I think it makes sense to move that to a variable!